### ISSUE TITLE
GEM Beamscraper: Fix system tests

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#3120]
The IOC system tests are failing on the GEM beamscraper jaws. As a developer I would like them fixed.
